src/bentoml_io/config.py

```diff
@@ -25,9 +26,9 @@ class TrafficSchema(TypedDict, total=False):
 
 
 class ResourceSchema(TypedDict, total=False):
-    num_cpus: Posint
```

Comment:
Oh, do the name and meaning change? Then `bentoml_io/schedule.py` also needs to be updated.